spans_buffer.go

```diff
@@ -1,57 +1,74 @@
 package lightstep
 
-import "github.com/opentracing/basictracer-go"
+import (
+	"time"
 
-const defaultMaxSpans = 1000
+	"github.com/opentracing/basictracer-go"
+)
 
 type spansBuffer struct {
-	rawSpans      []basictracer.RawSpan
-	maxBufferSize int
+	rawSpans       []basictracer.RawSpan
+	dropped        int64
+	reportOldest   time.Time
+	reportYoungest time.Time
```

Comment:
I'm on the bcronin boat of not being a fan of oldest/youngest. Perhaps `s/reportOldest/reportStart` and  `s/reportYoungest/reportEnd`?